cubedash/summary/_stores.py

```diff
@@ -128,18 +128,19 @@ def refresh_product(
     ):
         our_product = self.get_product_summary(product.name)
 
+        _LOG.info(f"Refreshing if {our_product.last_refresh_age} is older than {refresh_older_than}")
         if (
             our_product is not None
             and our_product.last_refresh_age < refresh_older_than
         ):
             _LOG.debug(
-                "init.product.skip.too_recent",
+                "init.product.skipping.too_recent",
```

Comment:
Most of the log codes use the third-person form, i'd like to keep them consistent unless we want to change all of them.

eg.https://github.com/opendatacube/datacube-explorer/blob/05fbeb731e6bd238c1646b9fed2934113e1b952b/cubedash/summary/_stores.py#L622
https://github.com/opendatacube/datacube-explorer/blob/05fbeb731e6bd238c1646b9fed2934113e1b952b/cubedash/summary/_stores.py#L630
https://github.com/opendatacube/datacube-explorer/blob/05fbeb731e6bd238c1646b9fed2934113e1b952b/cubedash/summary/_stores.py#L628
https://github.com/opendatacube/datacube-explorer/blob/591499d1377b41ed55079a5167caa31050f98fbd/cubedash/generate.py#L32